### PR TITLE
libcouchbase 2.5.2

### PR DESCRIPTION
**Pull request by mnunberg**

### CODE REVIEW

File: Library/Formula/libcouchbase.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Libcouchbase < Formula
   desc "C library for Couchbase"
   homepage "http://docs.couchbase.com/developer/c-2.4/c-intro.html"
-  url "https://s3.amazonaws.com/packages.couchbase.com/clients/c/libcouchbase-2.5.1.tar.gz"
-  sha1 "11a19b7dc7b8cad5ef6ad8a089d7a177c96451af"
+  url "https://s3.amazonaws.com/packages.couchbase.com/clients/c/libcouchbase-2.5.2.tar.gz"
+  sha1 "fa40e1637a245b08d3b2601bd4a021eb5b9881fa"
```
Comment (DomT4) [inline Library/Formula/libcouchbase.rb]: You'll need to use `sha256` here. We've deprecated SHA1.

**Comment by mnunberg** [comment]
Re-pushed with sha256

**Comment by mnunberg** [comment]
re-pushed with sha256

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/9664a0d16129c482b366e23d0d2750a8e700b56b. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

**Pull request closed without merging**